app/models/media_reaction.rb

```diff
@@ -0,0 +1,79 @@
+# == Schema Information
+#
+# Table name: media_reactions
+#
+#  id               :integer          not null, primary key
+#  media_type       :string           not null, indexed => [media_id, user_id]
+#  progress         :integer          default(0), not null
+#  reaction         :string(140)
```

Comment:
for what we were talking about in slack, we could technically have a `reaction_type` enum with `%i[individual, whole]` or something, but this idea would need to be approved by @JoshFabian